kusto/ingest/ingest.go

```diff
@@ -225,15 +227,15 @@ func (i *Ingestion) Stream(ctx context.Context, payload []byte, format DataForma
 	return err
 }
 
-func (i *Ingestion) getStreamConn() (*conn.Conn, error) {
+func (i *Ingestion) getStreamConn() (streamIngestor, error) {
 	i.connMu.Lock()
 	defer i.connMu.Unlock()
 
 	if i.streamConn != nil {
 		return i.streamConn, nil
 	}
 
-	sc, err := conn.New(i.client.Endpoint(), i.client.Auth(), i.client.HttpClient(), i.client.ClientDetails())
+	sc, err := kusto.NewConn(strings.Replace(i.client.Endpoint(), "ingest-", "", 1), i.client.Auth(), i.client.HttpClient(), i.client.ClientDetails())
```

Comment:
Another PR leaking in or is it intentional?

Reply:
I moved it to a function